src/services/itemMembership/service.ts

```diff
@@ -110,13 +110,15 @@ export class ItemMembershipService {
     actor: Account,
     repositories: Repositories,
     membership: { permission: PermissionLevel; itemId: UUID; memberId: UUID },
+    throwOnForbiddenPermission?: boolean,
```

Comment:
I see you start to need these a lot, should we try to move all `validatePermission` in the controller? to avoid creating many booleans to bypass verification.

Reply:
To easily solve that, I could ask an item id/path. I prefer to wait the refactor of ItemMembership before fixing it that way.

Otherwise, moving all the verification in the controller need big changes. I've made an Issue for that #1371, I'll fix it in another PR. Thank you for the feedback !